ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/deduplication/CommandDeduplicationBase.scala

```diff
@@ -469,7 +470,49 @@ private[testtool] abstract class CommandDeduplicationBase(
 
 object CommandDeduplicationBase {
   trait DelayMechanism {
-    def apply(): Future[Unit]
+    val deduplicationDuration: Duration
+    val extraWait: Duration
+
+    /** Delay by the guaranteed full deduplication period. After calling this method any duplicate calls should succeed
```

Comment:
This is not guaranteed. A ledger may extend the deduplication period arbitrarily, even beyond the max deduplication period in the time model (and IIUC this can actually happen with this change in kvutils).

Reply:
That is correct. I Will remove this comment but still leave the assumption in the test that this delay brings us "past" the deduplication period. Once we get the current changes in, and we re-evaluate the testing for command deduplication, we should either add metadata which helps us understand when the deduplication period ends/ provide a way to indicate to the test tool how it should calculate when the deduplication period ends/make the assertion for end of deduplication period optional

Reply:
Fine for now.